### ISSUE TITLE
vignette examples do not work

### ISSUE THREAD

**Issue by petermeissner** [cbailiss/pivottabler#1]
(Thanks for the good work!)

I was just trying out the version on CRAN and noticed that the examples using the `pivottabler()` function do not work: 

For Example for this vignette: https://cran.r-project.org/web/packages/pivottabler/vignettes/shiny.html the example reads: 

```r

  output$pvt <- renderPivottabler({
    pt <- PivotTable$new()
    pt$addData(bhmtrains)
    pt$addColumnDataGroups(input$selectCols)
    pt$addRowDataGroups(input$selectRows)
    pt$defineCalculation(calculationName="TotalTrains", summariseExpression="n()")
    pt$evaluatePivot()
    pivottabler(pt)
  })
```

... but actually should be this:

```r
  output$pvt <- renderPivottabler({
    pt <- PivotTable$new()
    pt$addData(bhmtrains)
    pt$addColumnDataGroups(input$selectCols)
    pt$addRowDataGroups(input$selectRows)
    pt$defineCalculation(calculationName="TotalTrains", summariseExpression="n()")
    pt$evaluatePivot()
    pt$renderPivot()
  })
```

... since the vignettes in the current version on Github do use tha latter anyway this post is more for reference if othe people stumble upon the same problem.

**Comment by cbailiss** [comment]
Thanks for the bug report.  

`pt$renderPivot()` and `pivottabler(pt)` are similar.  I'd intended that `pivottabler(pt)` was the function to use for Shiny.  
From looking at file history it seems that just before releasing to CRAN I removed the export for `pivottabler()` from the NAMESPACE file.  This is now corrected on Github, so the examples in the Shiny vignette should work again in the current version.
I'll keep this issue open until the version on CRAN is updated.

TLDR:  Either install the version from Github and use the vignette examples as-is, or if using the CRAN version, use the workaround described in the bug report above.
